zhaquirks/develco/power_plug.py

```diff
@@ -0,0 +1,106 @@
+"""Smart Plugs."""
+from zigpy.profiles import zha
+from zigpy.quirks import CustomCluster, CustomDevice
+from zigpy.zcl.clusters.general import (
+    Alarms,
+    Basic,
+    DeviceTemperature,
+    Groups,
+    Identify,
+    OnOff,
+    Ota,
+    Scenes,
+    Time,
+)
+from zigpy.zcl.clusters.homeautomation import ElectricalMeasurement
+from zigpy.zcl.clusters.measurement import OccupancySensing
+from zigpy.zcl.clusters.smartenergy import Metering
+
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+from zhaquirks.develco import DEVELCO
+
+DEV_TEMP_ID = DeviceTemperature.attributes_by_name["current_temperature"].id
+
+
+class DevelcoDeviceTemperature(CustomCluster, DeviceTemperature):
+    """Device Temperature. Modify divisor."""
+
+    def _update_attribute(self, attrid, value):
+        if attrid == DEV_TEMP_ID and value is not None:
```

Comment:
Did you ever get `None` as a value for `_update_attribute`?

Reply:
No, that check might be unnecessary.....

Reply:
Removed it for now.